server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -648,15 +650,15 @@ export function configureWebSocketServices(
 										error,
 									);
 								});
-						} catch (error) {
-							const errorMsg = `broadcast-signal conent body is malformed`;
-							return handleServerError(
-								logger,
-								errorMsg,
-								claims.documentId,
-								claims.tenantId,
-							);
 						}
+					} catch (error) {
+						const errorMsg = `Broadcast-signal failed with: ${error}`;
+						return handleServerError(
+							logger,
+							errorMsg,
+							claims.documentId,
+							claims.tenantId,
+						);
```

Comment:
We need to replace the `return` here with just a Lumberjack logging, right? And I think we can pass the error object directly to that call, so the additional message can just be "Unexpected error handling broadcastSignal event".